qa/rpc-tests/zkey_import_export.py

```diff
@@ -33,19 +34,16 @@ def setup_network(self, split=False):
     def run_test(self):
         [alice, bob, charlie, david, miner] = self.nodes
 
+        # the sender loses 'amount' plus fee; to_addr receives exactly 'amount'
         def z_send(from_node, from_addr, to_addr, amount):
-            opid = from_node.z_sendmany(from_addr, [{"address": to_addr, "amount": Decimal(amount)}])
+            global fee
+            opid = from_node.z_sendmany(from_addr,
+                [{"address": to_addr, "amount": Decimal(amount)}], 1, fee)
             wait_and_assert_operationid_status(from_node, opid)
             self.sync_all()
             miner.generate(1)
             self.sync_all()
 
-        def z_getbalance(node, zaddr):
-            bal = node.z_getbalance(zaddr)
-            # Ignore fees for sake of comparison
-            round_balance = math.ceil(bal*100)/100
```

Comment:
Removal of this causes the `math` module to become unused, triggering a PyFlakes warning and preventing merge.

Reply:
Fixing now, i see: ```pyflakes zkey_import_export.py 
zkey_import_export.py:12: 'math' imported but unused
zkey_import_export.py:52: undefined name 'cmp'
zkey_import_export.py:106: undefined name 'reduce'
```